### ISSUE TITLE
[semver:patch] Blocking Forever on NULL Job

### ISSUE THREAD

**Issue by andrew-barnett** [eddiewebb/circleci-queue#89]
### Orb version

1.8.1

### What happened

"Oldest job" and "Oldest Workflow Timestamp" are returning `null`, which then blocks forever. Could this possibly be related to #88?

```
fix/repo#3508 queueable
DEBUG: Making API Call to https://circleci.com/api/v2/me
DEBUG: API Success
Using API key for user: "xxx"
This build will block until all previous builds complete.
Max Queue Time: 10 minutes.
Only blocking execution if running previous jobs on branch: fix/repo#3508
Attempting to access CircleCI api. If the build process fails after this step, ensure your CIRCLECI_API_KEY  is set.
DEBUG: Making API Call to https://circleci.com/api/v1.1/project/github/xxx/xxx/tree/fix/repo#3508?filter=running
DEBUG: API Success
API access successful
Orb parameter block-workflow is true.
This job will block until no previous workflows have *any* jobs running.
Oldest job: null
This Workflow Timestamp: 
Oldest Workflow Timestamp: null
This build (2260) is queued, waiting for build number (null) to complete.
Total Queue time: 0 seconds.
Only blocking execution if running previous jobs on branch: fix/repo#3508
Attempting to access CircleCI api. If the build process fails after this step, ensure your CIRCLECI_API_KEY  is set.
DEBUG: Making API Call to https://circleci.com/api/v1.1/project/github/trade-platform/lib/tree/fix/repo#3508?filter=running
DEBUG: API Success
API access successful
Orb parameter block-workflow is true.
This job will block until no previous workflows have *any* jobs running.
Oldest job: null
This Workflow Timestamp: 
Oldest Workflow Timestamp: null
This build (2260) is queued, waiting for build number (null) to complete.
Total Queue time: 11 seconds.
```

My `config.yaml` looks like:

```yaml
orbs:
  queue: eddiewebb/queue@1.8.1

workflows:
  version: 2
  CircleCI:
    jobs:
      - queue/block_workflow:
          time: '10'
```

### Expected behavior

Things were working fine yesterday, but today it blocks until failure even if there is nothing else running.

**Comment by andrew-barnett** [comment]
Some debugging:

`curl -s -L  -H "Circle-Token: xxx" -H "Content-Type: application/json" 'https://circleci.com/api/v1.1/project/github/xxx/xxx/tree/fix/repo#3508?filter=running'` returns `[ ]`, even when there is a job running on that branch.

`curl -s -L  -H "Circle-Token: xxx" -H "Content-Type: application/json" 'https://circleci.com/api/v1.1/project/github/xxx/xxx/tree/fix/repo%233508?filter=running'` returns the full JSON of data while the job is running.

tl;dr: need to url encode the branch name; I'll put together a PR.
